subjects/DPxAI/declare-everything/README.md

```diff
@@ -0,0 +1,111 @@
+## Declare Everything
+
```

Comment:
Should we add the "mode" of the exercise here, like in the other quest exercises?